src/plugins/opencl_support/src/OpenCLSupportPlugin.cpp

```diff
@@ -36,8 +36,10 @@
 namespace U2 {
 
 extern "C" Q_DECL_EXPORT Plugin *U2_PLUGIN_INIT_FUNC() {
-    OpenCLSupportPlugin *plug = new OpenCLSupportPlugin();
-    return plug;
+    // Temporary disable OpenCL plugin
+    //OpenCLSupportPlugin *plug = new OpenCLSupportPlugin();
+    //return plug;
+    return nullptr;
```

Comment:
It's a bad behavior, error notification will be shown in every UGENE run. Add return to the beginning of the OpenCLSupportPlugin constructor.